build.py

```diff
@@ -6,7 +6,6 @@
 import uuid
 import platform
 import utility
-import polib
 
```

Comment:
What's the problem with having the import here?

Reply:
it imports polib before installing dev dependencies.